api/models/transaction.js

```diff
@@ -33,4 +33,6 @@ const Transaction = sequelize.define('transaction', {
 
 Transaction.belongsTo(Employee, { foreignKey: 'employeeId' });
 
+Employee.hasOne(Transaction);
```

Comment:
@arrustamyan this line can't be moved because we will have error as in Employee.js we using Transactionand in Transaction.js we using Employee, so it depends on each other